content/parts/en/contributor-form.mdx

```diff
@@ -0,0 +1,23 @@
+## Get credit for your contribution!
+
+Open Terms Archive is an Open Source initiative that relies on contributors just like you.
+
+We thus want to give you **full credit** for your contribution and in order to do that we need some small information from you.
+
+2 options now:
+
+### You want to take credit
+
+Awesome! Please, enter either of the following in the field below:
+
+- Your GitHub email address → You will be <a href="https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors" target="_blank">listed as a co-author</a>
+- Any email address → This address will be visible on the change proposed on GitHub
+
+<Input />
+<UpdateButton>Save email</UpdateButton>
+
+---
+
+### You don't want to take credit
+
+That's fine, just <IgnoreButton>click here</IgnoreButton>.
```

Comment:
```suggestion
## Sign your contribution

Open Terms Archive is an open source initiative that relies on contributors just like you 🤗 By default, your contributions will be anonymous. Signing your contributions with your email address will enable you to be notified when they are integrated or discussed and to be credited publicly for them.

<Input />
<UpdateButton>Sign your contribution with your email address</UpdateButton>

- While not readable anywhere on the web, this email address will be [publicly](https://stackoverflow.com/a/897604/594053) and irreversibly associated with your contribution.
- We will use this address only if it is necessary to contact you for integrating, crediting or otherwise handling this specific contribution.
- If you have an account on [GitHub](https://github.com), we recommend using an address linked to it so your account is mentioned.

<small>If you only want to contribute anonymously, you can <IgnoreButton>opt-out now</IgnoreButton>. You can always join the discussion later, but we will not be able to proactively contact you if we have questions.</small>
```